toolkit/resources/imageconfigs/packagelists/iso-initrd-packages.json

```diff
@@ -48,7 +48,6 @@
         "openssl",
         "azurelinux-release",
         "parted",
-        "pcre",
```

Comment:
do we need pcre2 here?

Reply:
Does not seem to be needed; I installed and booted the full iso from this build with no issues seen